hercules/py_sims.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, input_dict):
     def get_py_sim(self, py_sim_obj_dict):
         if py_sim_obj_dict["py_sim_type"] == "SimpleSolar":
             return SimpleSolar(py_sim_obj_dict, self.dt)
+        
```

Comment:
Is this section in the files twice?

Reply:
It looks like yes. Maybe this happened when I pulled in changes from develop? Not sure, but I'm fixing it now.